openmicroscopy/openmicroscopy
# Remove deprecated methods

# What this PR does

Removes the deprecated methods from the Gateway and their usage in Insight.

# Testing this PR

- Check integration tests.
- Perform some basic/general tasks in Insight and make sure nothing's broken.
- Check the changes in Polyline and Polygon Pojos. I'm not sure about the impact of these. Before the constructor and certain methods took three points parameters and one mask parameter, now it's just one points parameter. Has that really never been used before?
